onnxruntime/python/tools/quantization/onnx_quantizer.py

```diff
@@ -1098,7 +1098,8 @@ def quantize_weight_per_channel(
         reshape_dims = list(weights.shape)  # deep copy
         reshape_dims[channel_axis] = 1  # only one per channel for reshape
         quantized_weights = np.asarray(quantized_per_channel_data_list[0]).reshape(reshape_dims)
-        for i in range(1, len(quantized_per_channel_data_list)):
+        import tqdm
```

Comment:
This change would require a new dependency. Is it possible to use logging instead?